src/perception/se3.cpp

```diff
@@ -1,11 +1,13 @@
 #include "se3.hpp"
 
+#include <utility>
+
 /**
  *
  * @param position
  * @param rotation
  */
-SE3::SE3(Eigen::Vector3d const& position, Eigen::Quaterniond const& rotation) : position(position), rotation(rotation) {
+SE3::SE3(Eigen::Vector3d position, Eigen::Quaterniond const& rotation) : position(std::move(position)), rotation(rotation) {
```

Comment:
honestly this is a reasonably fast deep copy, I think performing the move here will cause more confusion than its worth.

Reply:
Tbh I don't even know why it is a move? It should be bit-copyable

Reply:
Figuring out later